package.json

```diff
@@ -61,64 +63,74 @@
         "lines": 95,
         "statements": 95
       }
-    }
+    },
+    "setupTestFrameworkScriptFile": "./test/setup.js"
   },
   "dependencies": {
+    "create-react-context": "^0.2.3",
     "hoist-non-react-statics": "^2.5.5",
     "intl-format-cache": "^2.0.5",
     "intl-messageformat": "^2.1.0",
     "intl-relativeformat": "^2.1.0",
-    "invariant": "^2.1.1"
+    "invariant": "^2.1.1",
+    "react-display-name": "^0.2.4",
+    "react-lifecycles-compat": "^3.0.4",
+    "rollup-plugin-virtual": "^1.0.1"
```

Comment:
Should this be a devDependency?

Reply:
The rollup plug in is needed to build the language data, which every user currently needs to do locally - so atm it shouldn't.

Reply:
You're right, I moved it.